impl/spanner/mutations/table_test.go

```diff
@@ -139,7 +143,7 @@ func TestReadLog(t *testing.T) {
 		low, high water.Mark
 		want      int
 	}{
-		{desc: "read nothing", low: ts1, high: ts1, want: 0},
+		// {desc: "read nothing", low: ts1, high: ts1, want: 0}, // Not supported by Cloud Spanner
```

Comment:
Delete entire line?

Reply:
I'll keep it around for when they implement this "feature"